content/en/data_streams/dotnet.md

```diff
@@ -27,12 +31,20 @@ environment:
   - DD_DATA_STREAMS_ENABLED: "true"
 ```
 
+### Monitoring connectors
```

Comment:
can we move this Connectors section to the bottom so it's the last section for every language setup page